sources/util.go

```diff
@@ -244,3 +246,39 @@ func mustParseInt(value string) int {
 }
 
 var accept = "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8"
+
+func unzipInMemory(ctx context.Context, resp *http.Response) []byte {
```

Comment:
This belongs to megaproxy file, because util has shared functions between all sources

Reply:
but can be useful in future to others that has zipped sources no? but if you prefer i can move

Reply:
Once we have other zipped sources - we'll move